docs/versions.md

```diff
@@ -49,8 +49,33 @@ So generally speaking, consider the following guidelines:
 
 * You should target the lowest version you get away with.
 * If you target .NET Standard 1.x, add .NET Standard 2.0 as [another target][multi-target].
-  This makes your library take advantage of the enhancements in the latest version
-  of different .NET platforms, such as .NET Core.
+
+The benefits of targeting .NET Standard 2.0 can be summarized as:
+
+* The dependency graph for .NET Standard 2.0 dependencies is greatly simplified.
+* Consumers of the package running on .NET Standard 2.0 compatible frameworks
+  will need to download less package dependencies as a result.
```

Comment:
`less` → `fewer` since `dependencies` is a countable noun.

Reply:
fixed